composer.json

```diff
@@ -15,26 +15,26 @@
         }
     },
     "require": {
-        "php": "^7.2 || ^8.0",
```

Comment:
Actually, the preferred syntax according to [Composer documentation](https://getcomposer.org/doc/articles/versions.md#version-range) for logical OR is double pipe (`||`). The single one is still valid for historical reasons.

Just add your actual changes and let's not fix the discrepancies between single and double pipe with this PR.